stable/postgresql/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if and .Values.metrics.enabled .Values.metrics.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "postgresql.fullname" . }}
+  labels:
```

Comment:
The namespace to deploy the ServiceMonitor should be configurable. See https://github.com/helm/charts/blob/master/stable/mariadb/templates/servicemonitor.yaml#L7

Reply:
Done